src/main/java/org/folio/event/util/KafkaEventUtil.java

```diff
@@ -27,4 +28,9 @@ public static String extractValueFromHeaders(List<KafkaHeader> headers, String k
       .orElse(null);
   }
 
+  public static Map<String, String> getHeaderMap(List<KafkaHeader> headers) {
```

Comment:
Changed to `TreeMap` to support case insensitive key, previous implementation return null of `X-Okapi-Tenant` header key (Kafka headers use lowercased header keys). For example: 

`{
	"folio.tenantId": "diku",
	"x-okapi-token": "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJVTkRFRklORURfVVNFUl9fMTcyLjE3LjAuMTo1NTE0NF9fMjAyNC0xMC0wN1QwNDo1NzoxMi41NzgrMDAwMCIsInR5cGUiOiJtb2R1bGUiLCJleHRyYV9wZXJtaXNzaW9ucyI6WyJTWVMjbW9kLWludmVudG9yeS1zdG9yYWdlLTI3LjIuMC1TTkFQU0hPVC4xMDMzIy9fL3RlbmFudCNbUE9TVF0iXSwidGVuYW50IjoiZGlrdSIsIm1vZHVsZSI6Im1vZC1pbnZlbnRvcnktc3RvcmFnZS0yNy4yLjAtU05BUFNIT1QuMTAzMyJ9.Bd7trhMwtdWN2CUnT9mBZ5p59EYOudlG1QPaYeE3e6k",
	"x-okapi-url": "http://host.docker.internal:9130",
	"x-okapi-tenant": "diku",
	"x-okapi-request-id": "936415/proxy;487539/tenant"`